roles/mesos/tasks/follower.yml

```diff
@@ -78,6 +78,15 @@
   tags:
     - mesos
 
+- name: set follower resources
+  sudo: yes
+  copy:
+    dest: /etc/mesos-slave/resources
+    content: "{{ mesos_resources }}"
+  notify: restart mesos follower
```

Comment:
Based on what I'm seeing on my instances I've got running:

`/tmp/mesos/meta` has boot_id (file) and then slaves (directory), inside of slaves is a link to the latest another directory within slaves that contains slave_info (not plain text) which seems to include a key "node_id". 

Based on that, it would seem `/tmp/mesos/meta/slaves/latest` holds the data you referenced.